implementations/exec.py

```diff
@@ -0,0 +1,59 @@
+from datetime import datetime
+
+from code_df import (
+    code_changes_git,
+    code_changes_lines,
+    reviews,
+    reviews_accepted,
+    )
+
+from scripts import (
+    code_changes_git,
+    code_changes_lines,
+    reviews,
+    reviews_accepted)
+
+import utils
+
+
+Git_metrics = [code_changes_git, code_changes_lines]
+GitHub_metrics = [reviews, ]
+
+
+def write_results(file=None, Git_results=None, GitHub_results=None):
+    print(Git_results)
+    print(GitHub_results)
+
+
+def test_git_metrics(items_file, date_range, is_code=None, conds=None):
```

Comment:
Why the method is called `test..`?

Reply:
It would be better if the method receives a list/generator of files instead of using `utils.read_json_file(items_file)`, thus the method will be really focused on calculating the metrics. The JSON files could be read in `main`, and this would help replacing the JSON files with a call to Perceval. WDYT?

Reply:
Good idea! Will change that.